### ISSUE TITLE
Undefined index: scope_closer

### ISSUE THREAD

**Issue by snapshotpl** [slevomat/coding-standard#902]
After update my code to 7.4 by Rector phpcs throw some internal exception.

```
Fatal error: Uncaught PHP_CodeSniffer\Exceptions\RuntimeException: Undefined index: scope_closer in /var/www/vendor/slevomat/coding-standard/SlevomatCodingStandard/Helpers/UseStatementHelper.php on line 195 in /var/www/vendor/squizlabs/php_codesniffer/src/Runner.php:606
Stack trace:
#0 /var/www/vendor/slevomat/coding-standard/SlevomatCodingStandard/Helpers/UseStatementHelper.php(195): PHP_CodeSniffer\Runner->handleErrors(8, 'Undefined index...', '/var/www/vendor...', 195, Array)
#1 /var/www/vendor/slevomat/coding-standard/SlevomatCodingStandard/Helpers/UseStatementHelper.php(137): SlevomatCodingStandard\Helpers\UseStatementHelper::getUseStatementPointers(Object(PHP_CodeSniffer\Files\LocalFile), 0)
#2 /var/www/vendor/slevomat/coding-standard/SlevomatCodingStandard/Sniffs/Namespaces/UselessAliasSniff.php(40): SlevomatCodingStandard\Helpers\UseStatementHelper::getFileUseStatements(Object(PHP_CodeSniffer\Files\LocalFile))
#3 /var/www/vendor/squizlabs/php_codesniffer/src/Files/File.php(496): SlevomatCodingStandard\Sniffs\Names in /var/www/vendor/squizlabs/php_codesniffer/src/Runner.php on line 606
```

**Comment by kukulich** [comment]
@snapshotpl What version do you use? What code does throw this error?

**Comment by snapshotpl** [comment]
v6.1.5. I don't know which file cause this error. I don't see any useful details in output to find it out.

**Comment by kukulich** [comment]
You can use verbose options for PHPCS.

**Comment by snapshotpl** [comment]
Got it! This is simplified version but still causes error:

```
<?php

class MyClass
{
    public function __construct() {
        if (
            $this->valid(fn(): bool => 2 > 1)
        ) {

        }
    }

    private function valid(callable $callable): bool
    {
        return $callable();
    }
}
```

**Comment by kukulich** [comment]
@snapshotpl Thank you for your report. However, it's a bug of PHPCS itself. Please report it directly to PHPCS. The `T_CLASS` token does not have scope pointers information.

**Comment by kukulich** [comment]
@snapshotpl I've prepared a workaround for now: https://github.com/slevomat/coding-standard/commit/4c88f6c3d81d189bf2f4dfb0946f145d816a2048
